CHANGELOG_UNRELEASED.md

```diff
@@ -10,6 +10,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- in `ssrnat.v`, added lemma `sumnB`.
```

Comment:
Should this be as follows?
```suggestion
- in `bigop.v`, added lemma `sumnB`.
```

Reply:
I did clone the repository and tried to animate the proof.  I went first to ssrnat (because of the CHANGELOG information) and then thought "I am stupid, this is obviously a bigop operation).  I spent so much time studying the code that I did not realize the mishap with not finding sumnB in ssrnat was really a bug in the changelog.  I agree with Anton, this needs to be changed.